build-support/bin/cherry_pick.sh

```diff
@@ -25,13 +27,11 @@ if [[ -z $PR_NUM ]]; then
   read -r PR_NUM
 fi
 
-TARGET_MILESTONE=$2
```

Comment:
The milestone changes should've been made on the "every milestone" PR, sorry y'all